oppijanumerorekisteri-service/src/main/java/fi/vm/sade/oppijanumerorekisteri/controllers/HenkiloController.java

```diff
@@ -231,6 +233,24 @@ public String createHenkiloFromHenkiloCreateDto(@ApiParam("Henkilön sukupuolen
         return this.henkiloModificationService.createHenkilo(henkilo).getOidHenkilo();
     }
 
+    @ApiOperation(value = "Henkilön olemassaolon tarkistus",
+            notes = "Tarkistaa henkilön olemassa olon annetun syötteen pohjalta.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Henkilölle löytyi oppijanumero", response = ExistenceCheckResult.class),
+            @ApiResponse(code = 204, message = "Henkilö on olemassa muttei oppijanumerorekisterissä", response = ExistenceCheckResult.class),
```

Comment:
204 = no content?

Reply:
true, different status code is used to indicate different semantics. response object exists, yes, but it is "empty". Should it be removed altogether?